src/styles/watch.scss

```diff
@@ -0,0 +1,76 @@
+.stopwatch {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  position: relative;
+  margin: 0 auto;
+  width: 500px;
```

Comment:
The size of the stopwatch is not compliant with the task requirements. It must be 80vmin. Replace the width with: 'width: 80vmin;'.